network/src/proto/mempool.proto

```diff
@@ -5,14 +5,54 @@ syntax = "proto3";
 
 package mempool;
 
+import "mempool_status.proto";
 import "transaction.proto";
+import "vm_errors.proto";
 
 /* MempoolSyncMsg represents the messages exchanging between validators to keep
  * transactions in sync. The proto definition provides the spec on the wire so
  * that others can implement their mempool service in various languages.
  * Mempool service is responsible for sending and receiving MempoolSyncMsg
  * across validators. */
 message MempoolSyncMsg {
+  oneof message {
+    SubmitTransactionsRequest submit_transactions_request = 1;
+    SubmitTransactionsResponse submit_transactions_response = 2;
+  }
+}
+
+// ------------------------------------------------------------
+// ---------------- Submit transactions -----------------------
+// ------------------------------------------------------------
+// The request for transaction submission.
+message SubmitTransactionsRequest {
   bytes peer_id = 1;
   repeated types.SignedTransaction transactions = 2;
 }
+
+// The response for transaction submission.
+message SubmitTransactionResponse {
```

Comment:
does it have to be included in this diff?
I see it's not actually used yet
And content of message is a bit questionable (something that we may want to clean up)